web/react/components/sidebar_right_menu.jsx

```diff
@@ -6,6 +6,10 @@ var client = require('../utils/client.jsx');
 var utils = require('../utils/utils.jsx');
 
 export default class SidebarRightMenu extends React.Component {
+    componentDidMount() {
+        $('.sidebar--left .dropdown-menu').perfectScrollbar();
```

Comment:
Are we not trying to get rid of perfect scroll bar? And got rid of it from here for a reason?